blog/_posts/2017-09-15-Team-Alexa-update.md

```diff
@@ -0,0 +1,74 @@
+---
+title: The end of Sasha&Sasha's journey
+layout: post
+created_at: Thu September 15 2017
```

Comment:
Your blogpost is scheduled to be published on Wed September 20 2017. Please change it at all places.